server/protocols/webadmin-cli/src/main/java/org/apache/james/cli/domain/DomainCreateCommand.java

```diff
@@ -43,7 +43,8 @@
     @Override
     public Integer call() {
         try {
-            DomainClient domainClient = Feign.builder()
+            DomainClient domainClient = new FeignClientFactory(domainCommand.webAdminCli.jwt)
+                .builder()
                 .target(DomainClient.class, domainCommand.webAdminCli.jamesUrl + "/domains");
```

Comment:
We should have a method for making this easier: `domainCommand.fullyQualifiedURL("/domains")`